contracts/dtoken/src/common.rs

```diff
@@ -70,28 +70,29 @@ impl Contract {
 
     pub fn set_total_reserves(&mut self, amount: Balance) -> Balance {
         self.total_reserves = amount;
-        self.get_total_reserves()
+        self._get_total_reserves()
     }
 
     pub fn set_total_borrows(&mut self, amount: Balance) -> Balance {
         self.total_borrows = amount;
-        self.get_total_borrows()
+        self._get_total_borrows()
     }
-}
 
-#[near_bindgen]
-impl Contract {
-    pub fn get_total_supplies(&self) -> Balance {
+    pub fn _get_total_supplies(&self) -> Balance {
```

Comment:
@pahomas as I understand we don't neet underscore anymore because we use those methods in view module, 
or if this is done to resolve name conflict, let's call "view" methods `view_*` instead of `get_*`